platforms/jvm/plugins-java-base/src/main/java/org/gradle/api/plugins/JavaPluginExtension.java

```diff
@@ -279,6 +285,7 @@ public interface JavaPluginExtension {
      *
      * @since 7.1
      */
+    @ToBeKeptEagerProperty(because = "Read-only nested property")
```

Comment:
❓ This should not need an annotation as it's a `DomainObjectContainer`.

Reply:
Done, annotation removed, this won't be reported anymore